### ISSUE TITLE
Tapo device integration missing

### ISSUE THREAD

**Issue by robcsi** [petretiandrea/home-assistant-tapo-p100#560]
**Is your feature request related to a problem? Please describe.**
I'm unable to add integration to Tapo P115 plug. It does not appear in the integration list, or in the HACS list.

**Device Info**
I'm using HomeAssistant on an RPI400 install with rpi-imager directly as an OS on an SD card.
Home Assistant 2023.10.2
Supervisor 2023.10.0
Operating System 10.5
Frontend 20231005.0 - latest

**Comment by robcsi** [comment]
If I add the files in the "Manual way", as described on the main page of this repo, TP-link Tapo appears in the integration list, but theoretically I shouldn't add it if I understand correctly:
"This integration is part of HACS store, so you don't need anymore to add this repository as a custom repository. You can find it directly on HACS Store: search for tapo and you will find Tapo Controller. (HACS >= 1.6.0 is required)"

**Comment by robcsi** [comment]
I found the HACS website in the meantime and managed to install, using their installation script: https://hacs.xyz/

**Comment by petretiandrea** [comment]
Yes you need to install HACS, it's the recommended way and the simple one
